x-pack/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/status_action.tsx

```diff
@@ -189,6 +199,34 @@ export const EndpointStatusActionResult = memo<
           </ConsoleCodeBlock>
         ),
       },
+      {
+        title: (
+          <ConsoleCodeBlock>
+            {i18n.translate('xpack.securitySolution.endpointResponseActions.status.policyVersion', {
+              defaultMessage: 'Policy version',
+            })}
+          </ConsoleCodeBlock>
+        ),
+        description: (
+          <ConsoleCodeBlock>
+            {endpointDetails.metadata.Endpoint.policy.applied.version}
```

Comment:
This is not correct. The property to show for version is `endpoint_policy_version`:

https://github.com/elastic/kibana/blob/89b89abf20962b2609f4f8243c11c18c3c7f43d0/x-pack/plugins/security_solution/common/endpoint/types/index.ts#L523-L524

Sorry if this was not clear in the Issue I created.

Also - maybe place a `v` before the number? like, ex: `v6`